real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/RoboBuggyGUIContainer.java

```diff
@@ -0,0 +1,38 @@
+package com.roboclub.robobuggy.ui;
+
+import java.awt.Component;
+import java.awt.Container;
+import java.awt.Graphics;
+import java.util.ArrayList;
+
+import javax.swing.JPanel;
+
+public class RoboBuggyGUIContainer extends JPanel{
+	ArrayList<ComponentData> components = new ArrayList<ComponentData>();
+	String name = "unknown";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1746561348/issues/source?bid=2938665&fileBranchId=3059924#l12)